test/quantization/test_equalize.py

```diff
@@ -0,0 +1,100 @@
+import torch
+import torch.nn as nn
+
+from torch.testing._internal.common_quantization import QuantizationTestCase
+
+import torch.quantization._equalize as _equalize
+
+import copy
+
+class TestEqualizeEager(QuantizationTestCase):
+    def channels_equalized_test(self, tensor1, tensor2, output_axis, input_axis):
+        output_channel_tensor1 = _equalize.channel_range(tensor1, output_axis)
+        input_channel_tensor2 = _equalize.channel_range(tensor2, input_axis)
+
+        # ensuring the channels ranges of tensor1's input is the same as
+        # tensor2's output
+        self.assertEqual(output_channel_tensor1, input_channel_tensor2)
+
+    def get_module(self, model, name):
```

Comment:
same, this should be getModule

Reply:
also a helper function, do we still name it as test_xxx?

Reply:
no, it should be getModule if it's a helper function